styles.css

```diff
@@ -0,0 +1,11 @@
+button {
+    font-size: 25pt;
+}
+
+.correctButton {
+    background-color: #04ac04;
+}
+
+.wrongButton {
+    background-color: red;
+}
```

Comment:
Are you going to use these classes, or just change the colours in the javascript? Pick one so that it's less confusing!